The Solaris OVS Interface driver needs to be added to the list of entry points
in order to be discovered and loaded by stevedore.

*** neutron-8.0.0/setup.cfg	2016-04-07 00:47:18.000000000 -0700
--- new/setup.cfg	2016-05-24 13:31:12.688314864 -0700
***************
*** 155,161 ****
  	ivs = neutron.agent.linux.interface:IVSInterfaceDriver
  	linuxbridge = neutron.agent.linux.interface:BridgeInterfaceDriver
  	null = neutron.agent.linux.interface:NullDriver
! 	openvswitch = neutron.agent.linux.interface:OVSInterfaceDriver
  neutron.agent.firewall_drivers = 
  	noop = neutron.agent.firewall:NoopFirewallDriver
  	iptables = neutron.agent.linux.iptables_firewall:IptablesFirewallDriver
--- 155,161 ----
  	ivs = neutron.agent.linux.interface:IVSInterfaceDriver
  	linuxbridge = neutron.agent.linux.interface:BridgeInterfaceDriver
  	null = neutron.agent.linux.interface:NullDriver
! 	openvswitch = neutron.agent.solaris.interface:OVSInterfaceDriver
  neutron.agent.firewall_drivers = 
  	noop = neutron.agent.firewall:NoopFirewallDriver
  	iptables = neutron.agent.linux.iptables_firewall:IptablesFirewallDriver
